### PR TITLE
doc(gry): add acrobot zh doc

### PR DESCRIPTION
**Pull request by ruoyuGao**

### CODE REVIEW

File: source/13_envs/acrobot_zh.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+--------
+
+在 Python 命令行中运行如下命令验证安装成功。
+
+.. code:: shell 
+
+    import gym
+    env = gym.make('Acrobot-v1')
+    obs = env.reset()
+    print(obs)
```
Comment (PaParaZz1) [inline source/13_envs/acrobot_zh.rst:33]: add an assertion expression

File: source/13_envs/acrobot_zh.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+-  \ ``0的力矩`` \: 1 表示向关节施加0的力矩。
+
+-  \ ``+1的力矩`` \: 2 表示向关节施加+1的力矩。
+
+使用 gym 环境空间定义则可表示为：
+
+.. code:: python
+    
+    action_space = spaces.Discrete(3)
```
Comment (PaParaZz1) [inline source/13_envs/acrobot_zh.rst:53]: you should import `spaces` first or use `gym.spaces`

File: source/13_envs/acrobot_zh.rst [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+
+theta1 是第一个关节的角度，其中角度 0 表示第一个链接直接指向下方。
+
+theta2 是相对于第一个连杆的角度。 角度 0 对应于两个链接之间具有相同的角度。
+
+
+奖励空间
+-----------
+目标是让自由端以尽可能少的步数达到指定的目标高度，因此所有未达到目标的步数都会产生 -1 的奖励。 达到目标高度会导致终止，奖励为 0。
```
Comment (PaParaZz1) [inline source/13_envs/acrobot_zh.rst:81]: you can use bold to emphasize some keywords

File: source/13_envs/acrobot_zh.rst [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+
+终止条件
+------------
+Cartpole 环境每个 episode 的终止条件是遇到以下任何一种情况：
```
Comment (PaParaZz1) [inline source/13_envs/acrobot_zh.rst:86]: Cartpole?

File: source/13_envs/acrobot_zh.rst [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+            policy = DQNPolicy(cfg.policy, model=model)
+
+            task.use(interaction_evaluator(cfg, policy.eval_mode, evaluator_env))
+            task.use(eps_greedy_handler(cfg))
+            task.use(StepCollector(cfg, policy.collect_mode, collector_env))
+            task.use(nstep_reward_enhancer(cfg))
+            task.use(data_pusher(cfg, buffer_))
+            task.use(OffPolicyLearner(cfg, policy.learn_mode, buffer_))
+            task.use(online_logger(train_show_freq=10))
+            task.use(CkptSaver(policy,save_dir='/Your_path/acrobot_logs', train_freq=100))
```
Comment (PaParaZz1) [inline source/13_envs/acrobot_zh.rst:142]: format these code lines

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/13_envs/acrobot_zh.rst (hunk 1 above)
Comment (ruoyuGao) [reply source/13_envs/acrobot_zh.rst]: Added

File: source/13_envs/acrobot_zh.rst (hunk 2 above)
Comment (ruoyuGao) [reply source/13_envs/acrobot_zh.rst]: fixed

File: source/13_envs/acrobot_zh.rst (hunk 4 above)
Comment (ruoyuGao) [reply source/13_envs/acrobot_zh.rst]: typo mistake, check the whole article and changed it.

File: source/13_envs/acrobot_zh.rst (hunk 5 above)
Comment (ruoyuGao) [reply source/13_envs/acrobot_zh.rst]: follow the style of other docs. So I use **acrobot_dqn_config.py** to replace the demo code. Now I think the format now is good.

**Pull request merged**